Bfresco tutorial

ozgesurer opened the pull request:
Adding colab documents for BAND Camp 2021 to the repo and linking them through the RST files.

ozgesurer committed:
Create bandcamp2021_nbs

ozgesurer committed:
including nbs

ozgesurer committed:
Update BANDCamp21.rst

ozgesurer committed:
Update BANDCamp21.rst

ozgesurer committed:
Update BANDCamp21.rst

ozgesurer committed:
Update BANDCamp21.rst

ozgesurer committed:
Update README.rst

ozgesurer committed:
Update README.rst

ozgesurer committed:
Update README.rst

ozgesurer committed:
Update BANDCamp21.rst

wildsm reviewed (commented):
This PR addresses the issue with the previous colab-linked notebooks being no longer available. 

It moves the notebooks directly under version control in bandframework.

I have confirmed that the links work.

I plan to update the changelog to reflect these changes, but may wait until the changelog is updated from #181

DanielRPhillips commented:
I have also reviewed the links and confirmed they are correct.

wildsm commented:
Update: now waiting on #184

wildsm committed:
Merge pull request #186 from bandframework/develop

Bringing up to date with new develop

wildsm committed:
Update CHANGELOG 

Added new BAND-compatible tools and updated existing ones, including Bfrescox and various other packages.

wildsm reviewed (approved):
@ozgesurer These look good, but can you please double check my edit to the CHANGELOG?

ozgesurer commented:
@wildsm This looks good: Updated BAND software use case from BAND Camp 2021 to directly include Bfrescox tutorials. Do you want me to merge or wait?

wildsm commented:
Go for it!
